scripts/lint.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+pylint certbot_dns_loopia.py
```

Comment:
This script assumes you're running it from the repository root, so if you cd into scripts and run it it will not work.

Reply:
Yes, that's right. I suppose I could check if the current directory is `scripts` and just do `cd ..` in that case. But that still leaves other scenarios failing, like if I am somewhere completely different (let's say `/tmp`) and try to run the script using an absolute path.

To me, there are two scenarios for running this script:

1. GitHub Action runs it and everything works since it runs from the project root
2. A user runs it from command line, and the project root seems like a rational location to run it from

Maybe the best path of action is to just print an informational message (`This script needs to be executed from the project root`) if run from anywhere else than the root?

Reply:
You can do

```bash
cd "$(dirname "$0")"
```

at the top of your script.

Reply:
Clever! I have added this.